gnat2goto/driver/tree_walk.adb

```diff
@@ -1364,22 +1410,27 @@ package body Tree_Walk is
    -----------------------
 
    function Do_Loop_Statement (N : Node_Id) return Irep is
+      Iter_Scheme  : constant Node_Id := Iteration_Scheme (N);
+      Body_Block   : constant Irep := Process_Statements (Statements (N));
 
-      Iter_Scheme : constant Node_Id := Iteration_Scheme (N);
-      Body_Block  : constant Irep := Process_Statements (Statements (N));
+      function Do_For_Statement (Param : Irep; Cond : Irep; Post : Irep)
+                                 return Irep;
 
-      function Do_For_Statement return Irep;
       function Do_While_Statement (Cond : Irep) return Irep;
 
       ----------------------
       -- Do_For_Statement --
       ----------------------
 
-      function Do_For_Statement return Irep is
+      function Do_For_Statement (Param : Irep; Cond : Irep; Post : Irep)
+                                 return Irep is
```

Comment:
style: line break before "is" (if in doubt, check sem_util.adb, which is big enough to contain all possible variants of formatting)